src/api/app/services/consistency_check_job_service/backend_package_importer.rb

```diff
@@ -0,0 +1,39 @@
+module ConsistencyCheckJobService
+  class BackendPackageImporter
+    attr_reader :errors
+
+    def initialize(project, package_name)
+      @project = project
+      @package = @project.packages.build(name: package_name)
+      @errors = []
+    end
+
+    def call
+      create_package_frontend
+    rescue ActiveRecord::RecordInvalid,
+           Backend::NotFoundError
+      delete_package
+      @errors << delete_error_message
+    end
+
+    private
+
+    def delete_error_message
+      "DELETED in backend due to invalid data #{@project.name}/#{@package.name}"
+    end
+
+    def create_package_frontend
```

Comment:
@vpereira Just a suggestion, I would name it `create_package_in_frontend`, makes it more clear on the first read